### ISSUE TITLE
Question on pose data

### ISSUE THREAD

**Issue by malcolmgarner-movement** [NVIDIA/vid2vid#128]
Loving this work, it is amazing what has been accomplished in this field of research!

I have a question about the body to pose segment: how does this actually work? Are two videos taken and the actions of one video are transposed onto the actor in the second video?

I've read through the documentation and so far I haven't been able to determine the basic mechanic behind this specific incarnation.

Any assistance would be appreciated, and if there is material I might have missed, please don't hesitate to point me in the right direction.

Thanks in advance

**Comment by pranavraikote** [comment]
@malcolmgarner-movement Based on given DensePose mask frames + Input frames + OpenPose files, you can give your choice of target images, and it should generate something similar. This is as far as my understanding, yet to test it thoroughly with any permutations and combinations.

**Comment by git-hamza** [comment]
@pranavraikote I don't think it mentions anywhere about the target images. Doesn't it says that it synthesizes on videos that was learned (training videos)?
